### DISCUSSION TITLE
Widescreen Addon Idea

### DISCUSSION THREAD

**Discussion by GoldcatGames** [ScratchAddons/ScratchAddons#3868] [Ideas]
Maybe Add An Addon That Makes Scratch Have Widescreen, Basically A 16:9 Ratio.

**Comment by Cookiesmuch** [comment]
CURRENTLY this is possible using turbowarp by manually changing the stage size

**Reply by Cookiesmuch** [reply]
Please note that there might be no possible way to change stage aspect ratio without completely changing stage size

**Reply by Cookiesmuch** [reply]
Changing stage size is not available for anyone on scratch (they need to use turbowarp editor)

**Reply by Cookiesmuch** [reply]
and stage size DOES NOT save.

**Reply by Cookiesmuch** [reply]
however for purposes its easy enough to change stage size to fit your screen ratio, then use a full screen function (idk if that works in the turbowarp editor btw)

**Reply by Cookiesmuch** [reply]
In terms of finding a way to only change how it shows on the monitor, it seems impossible especially since how would sprites function outside of the usual stage size?

**Reply by Cookiesmuch** [reply]
so again turbo warpw ill probably be the only way.

**Reply by GoldcatGames** [reply]
oh ok

**Comment by Endermaster3690** [comment]
what about people who dont use SA

**Reply by GoldcatGames** [reply]
maybe ( this is what i would do) I would shrink the sprite sizes and add a sprite with the border
